manager/integration/tests/test_basic.py

```diff
@@ -1304,7 +1304,11 @@ def backup_labels_test(client, random_labels, volume_name, size=SIZE, backing_im
     # If we're running the test with a BackingImage,
     # check field `volumeBackingImageName` is set properly.
     backup = bv.backupGet(name=b.name)
-    assert len(backup.labels) == len(random_labels)
+    # Now Longhorn auto will auto create label Longhorn.io/volume-access-mode
+    # in backup
```

Comment:
```suggestion
    # Longhorn will automatically add a label `longhorn.io/volume-access-mode` to a newly created backup
```

Reply:
Hi @innobead , I updated comment above for review, thank you